plugins/knife/knife-box-create.rb

```diff
@@ -97,6 +105,24 @@ def run(supported_boxes = [])
       new_box = box_manager.create_box(role, {index: config[:index], ralf: ((role == "ralf") || config[:ralf]), seagull: config[:seagull]})
       instance_id = new_box.id
 
+      if config[:standalone]
+        # The box is a standalone box, so add it now to the etcd cluster,
+        # and create the shared configuration
+        boxes = find_nodes(chef_environment: config[:environment])
+        if boxes.size == 1
+          # Check that there's only one box in this environment; if there's
+          # more then we shouldn't be overriding any etcd settings
+          print "Box is standalone, so updating etcd configuration\n"
+          boxes[0].set[:clearwater][:etcd_cluster] = true
+          boxes[0].run_list << "role[shared_config]"
+          boxes[0].save
+          trigger_chef_client(config[:cloud],
+                              "chef_environment:#{config[:environment]}")
+        else
+          print "Can't use standalone on an existing deployment. No changes have been made to etcd configuration\n"
```

Comment:
Should we say "Use 'knife deployment resize' to enlarge your existing deployment?".